Uplift/Views/SportsDetail/SportsDetailCommentCollectionViewCell.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  SportsDetailCommentTableViewCell.swift
+//  Uplift
+//
+//  Created by Artesia Ko on 4/25/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class SportsDetailCommentCollectionViewCell: UICollectionViewCell {
+    
+    private var bubble = UIView()
+    private var commentLabel = UILabel()
+    private var nameLabel = UILabel()
+    private var profileImage = UIImageView()
+    private var timeLabel = UILabel()
+    
+    private let imageSize = 32
+
+     override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        profileImage.backgroundColor = .gray02
+        profileImage.contentMode = .scaleAspectFill
+        profileImage.layer.cornerRadius = CGFloat(imageSize/2)
```

Comment:
Can we make this `2.0`? `imageSize` gets typecast as an `int` so right now you're dividing two `ints` and are losing some decimal values.